RackApp/lib/myproject/app.rb

```diff
@@ -0,0 +1,97 @@
+# frozen_string_literal: true
+
+require 'erb'
+require 'rack'
+require 'active_support/time'
+require 'active_support/core_ext'
+class MyController
+  class_attribute :template_name, instance_writer: false
+  self.template_name = 'template'
+  attr_reader :person, :time
+  def initialize(user, time)
+    @person = user
+    @time = time
+  end
+
+  def template(format)
+    View.new(template_name, format)
+  end
+
+  def render_template(format)
+    template(format).render.result(binding)
+  end
+end
+
+class View
+  def initialize(name, format)
+    @erb = file(name, format)
+  end
+
+  def file(name, format)
+    file_path = File.expand_path(File.join('../../..', 'templates', "#{name}.#{format}.erb"), __FILE__)
+    if File.exist?(file_path)
+      File.read(file_path)
+    else
+      'ERROR:The file doesnot exist'
```

Comment:
raise an error instead ;)
Otherwise the result of the file will be displayed to the User
It is better to catch the error when it is really necessary and not assuming what it should do